trie/utils/binaries.py

```diff
@@ -0,0 +1,66 @@
+from trie.constants import (
+    TWO_BITS,
+    PREFIX_00,
+    PREFIX_100000,
+)
+
+
+def safe_ord(value):
+    if isinstance(value, int):
```

Comment:
FYI, `isinstance(value, int)` will return truthy for `True` and `False`.  You can use `eth_utils.is_integer` if you need that kind of precision.